hydra-client/src/main/java/com/pandora/hydra/client/HydraClient.java

```diff
@@ -68,6 +75,29 @@ private HydraApi createHydraApi(Configuration config) {
         return retrofit.create(HydraApi.class);
     }
 
+    /**
+     * Creates a Proxy object if the appropriate System properties were specified at JVM startup.
+     * The Java standard for proxy usage seems to be to pass -Dhttp.proxyPort and -Dhttp.proxyHost as args to the JVM.
+     *
+     * @return Proxy object if one could be created from the specified system properties; otherwise null
+     */
+    static Proxy createProxyIfSpecified() {
+        String proxyHost = System.getProperty("http.proxyHost");
+        if (proxyHost == null || proxyHost.isEmpty()) {
```

Comment:
I think breaking this into two statements would be more clear. Saying that no proxyHost is invalid is a little confusing in my opinion

Reply:
I think I addressed this now.